cypress/support/pages/home.pageObject.js

```diff
@@ -4,7 +4,17 @@ class HomePageObject extends PageObject {
   url = '/#/';
 
   get usernameLink() {
-    return cy.getByDataCy('username-link');
+    return cy.getByDataCy('usernameLink');
+  }
+
+  cheackArticlesList() {
+    cy.get('.article-preview')
+      .should('contain', 'No articles')
+  }
+
+  checkRegistration() {
+    cy.get('.swal-text')
```

Comment:
Similar to the previous error, you are using 'cy.get' method to get an element by its class name. This makes your tests fragile and dependent on CSS. Use 'cy.getByDataCy' method or similar one to get elements by their 'data-qa' attribute instead.